README.md

```diff
@@ -1,2 +1,142 @@
-# rise-logger
-Web component from Rise Vision used for logging usage of a parent web component
+# Rise Logger Web Component [![CircleCI](https://circleci.com/gh/Rise-Vision/rise-logger/tree/master.svg?style=svg)](https://circleci.com/gh/Rise-Vision/rise-logger/tree/master)
+
+## Introduction
+
+The `rise-logger` Web Component uses Google’s [Big Query](https://cloud.google.com/bigquery/) to log usage data of a parent Rise Vision web component (eg. [rise-google-sheet](http://rise-vision.github.io/rise-google-sheet)). 
+
+Rise Logger Web Component works in conjunction with [Rise Vision](http://www.risevision.com), the [digital signage management application](http://rva.risevision.com/) that runs on [Google Cloud](https://cloud.google.com).
+
+At this time Chrome is the only browser that this project and Rise Vision supports.
+
+## Usage
+To use the Rise Logger Web Component, you should first install it using Bower:
+```
+bower install https://github.com/Rise-Vision/rise-logger.git
+```
+
+Next, add an instance to your parent component in the dom-module. You should also include the source of  `rise-logger` before any code that executes in your parent component. For example:
```

Comment:
@stulees I was just wondering if it would work in a Widget as well. For our purposes it's fine and I don't think we need to add support for that at this point.